pallets/creator-staking/src/inflation.rs

```diff
@@ -0,0 +1,44 @@
+use crate::*;
+use frame_support::traits::{Currency, Get, OnTimestampSet, Imbalance};
+use sp_runtime::traits::{Saturating, SaturatedConversion, UniqueSaturatedInto, Zero};
+
+type NegativeImbalanceOf<T> = <<T as Config>::Currency as Currency<
+    <T as frame_system::Config>::AccountId,
+>>::NegativeImbalance;
+
+impl<Moment, T: Config> OnTimestampSet<Moment> for Pallet<T> {
+    fn on_timestamp_set(_moment: Moment) {
```

Comment:
Add an explainer comment about this method

Reply:
Why if we have docs?

https://paritytech.github.io/substrate/master/frame_support/traits/trait.OnTimestampSet.html